### PR TITLE
Add 404 handler for invalid tags

### PR DESCRIPTION
**Pull request by matdave**
A basic fix to the TaggerGetResourcesWhere snippet that optionally sends a 404 error response if an invalid tag is selected.

### CODE REVIEW

File: core/components/tagger/elements/snippets/taggergetresourceswhere.snippet.php [hunk 1]
```diff
@@ -30,6 +31,7 @@
 $tagField = $modx->getOption('tagField', $scriptProperties, 'alias');
 $likeComparison = (int) $modx->getOption('likeComparison', $scriptProperties, 0);
 $matchAll = (int) $modx->getOption('matchAll', $scriptProperties, 0);
+$errorOnInvalidTags = $modx->getOption('errorOnInvalidTags', $scriptProperties, false);
```
Comment (theboxer) [inline core/components/tagger/elements/snippets/taggergetresourceswhere.snippet.php:34]: You can't pass boolean as property from MODX. So I'd use something like:
```php
intval($modx->getOption('errorOnInvalidTags', $scriptProperties, 0)) === 1
```

**Comment by JoshuaLuckers** [comment]
@theboxer @matdave this would be really great to have!

**Comment by matdave** [comment]
@JoshuaLuckers you're telling me 😆 I've been working on some extras updates and will try to bundle this one up

File: core/components/tagger/elements/snippets/taggergetresourceswhere.snippet.php (hunk 1 above)
Comment (matdave) [reply core/components/tagger/elements/snippets/taggergetresourceswhere.snippet.php]: @theboxer I've updated this

**Pull request merged**